journal/modernized/36-chapter-33.adoc

```diff
@@ -322,22 +322,22 @@ to repent, believe, and obey the gospel;
 to turn to the light of Christ Jesus in his own heart, follow it, and live;
 he would then no longer walk in darkness, but in the light of life.
 Fakafonua interrupted me once or twice to express his thanks; and when I stopped,
-he told James Watkin again to thank me, and said, '`I know very well it is true.
+he told James Watkin again to thank me, and said, "`I know very well it is true.
 I have thought of these things, and understand,
 and am convinced that it is better to be a Christian; but I can never be one,
 I am too great a sinner to be saved:
-I must remain as I am.`' I told James Watkin to tell him,
+I must remain as I am.`" I told James Watkin to tell him,
 that as he knew and understood these things,
 his condemnation would be greater on that account;
 and certainly his destruction would be of himself.
 
 James Watkin then said he thought I had better speak to the sailors,
 who sat and listened with great attention, I replied,
-'`there is no occasion for that,--they know very well.`' James Watkin replied,
-'`he thought they did not.`' '`Yes,`' said I, '`there is not one of them, but knows very well,
+"`there is no occasion for that,--they know very well.`" James Watkin replied,
+"`he thought they did not.`" "`Yes,`" said I, "`there is not one of them, but knows very well,
```

Comment:
this `he thought they did not` is another instance where if we imagine this like a play, it reveals that it's not a direct quote, so again here maybe let's nix the quotes around just this phrase in the modernized, and keep it single in the original.